06-developing-with-invenio/README.md

```diff
@@ -114,7 +114,7 @@ Run tests again:
 
 In order to test a new package, simply install it in the virtualenv using `pip` tool:
 
-```bash
+```console
 (my-site)$ pip install Pillow
```

Comment:
If we decide to remove every `$` character to make the strings copy-pastable maybe we should to the same with the virtual env prompts (if no additional clarificaiton on it being used is required)

Reply:
I don't have any objection to that. 
But these prompts provide a hint to the reader that the command was run in a specific venv. Otherwise the command should be modified as well (e.g. in this case `pipenv run pip install Pillow`)